easybuild/easyblocks/t/tensorflow.py

```diff
@@ -778,12 +797,6 @@ def test_step(self):
 
     def install_step(self):
         """Custom install procedure for TensorFlow."""
-
-        # avoid that pip (ab)uses $HOME/.cache/pip
-        # cfr. https://pip.pypa.io/en/stable/reference/pip_install/#caching
-        env.setvar('XDG_CACHE_HOME', tempfile.gettempdir())
-        self.log.info("Using %s as pip cache directory", os.environ['XDG_CACHE_HOME'])
```

Comment:
This was added for a good reason, I don't think we should stop setting `$XDG_CACHE_HOME`?

Reply:
Yes. See the commit message: `Remove setting of XDG_CACHE_HOME (done in PythonPackage)` ;)

So seems this was moved up one EasyBlock. I can rebuild again just to double-check this is actually used

Reply:
From the log send in slack:
```
== 2020-09-15 17:31:46,045 pythonpackage.py:260 INFO Using default value for expected module name (lowercase software name): 'tensorflow'
== 2020-09-15 17:31:46,045 pythonpackage.py:296 INFO Using pip with --no-deps option
== 2020-09-15 17:31:46,046 environment.py:97 INFO Environment variable XDG_CACHE_HOME set to /import/ebswbuild/home/easybuild/builddirs/builddir.bdw/zao/tmpdir/9983760/eb-4s0xF9 (previous value: '/import/ebswbuild/home/easybuild/builddirs/builddir.bdw/zao/tmpdir/9983760/eb-4s0xF9')
```

So I'd say this verifies this